docs/learn-DOT.md

```diff
@@ -8,29 +8,16 @@ sidebar_label: DOT
 
 DOT is the native token of the Polkadot network in a similar way that BTC is the native token of Bitcoin or ether is the native token of the Ethereum blockchain.
 
-Alexander testnet DOTs have 15 decimal places, while the smallest unit is called the Planck. Kusama tokens (KSM) have 12 decimal places. You can compare the Planck to satoshis or wei, while the DOT is like a bitcoin or an ether.
-
-**The table show as below is the old version.**
-
-|Unit |Decimal Places|
-|-----|--------|
-|Femto| 0  |
-|Pico | 3  |
-|Nano | 6  |
-|Micro| 9  |
-|Milli| 12 |
-|DOT  | 15 |
-
-**New Version**
+Kusama tokens (KSM) and DOT tokens both have 12 decimal places, with the smallest unit being the Planck (a reference to (Planck Length)[https://en.wikipedia.org/wiki/Planck_length], the smallest possible distance in the physical Universe). You can compare the Planck to satoshis or wei, while the DOT is like a bitcoin or an ether.
 
+**Polkadot**
 |Unit |Decimal Places|
 | --- | --- |
 Planck| 1
 Point | 3
 Microdot (uDOT)| 6
 Millidot (mDOT)| 9
 Dot (DOT) | 12
```

Comment:
Does this now reflect the new decimal point move that's planned for mainnet, or is that still TBD?

Reply:
That's still TBD, and if it does occur, will be voted upon first.  If it passes, we can update it, but for now, this is the situation.

Reply:
I think this is resolved, feel free to un-resolve if not